test/Interop/Cxx/static/Inputs/static-local-var.h

```diff
@@ -0,0 +1,4 @@
+static inline int counter() {
+  static int c = 0;
+  return c++;
+}
```

Comment:
It would be nice to define a wrapper for `counter()` defined in a `.cc` file, and verify that the `static int c` is indeed shared between the copies of `counter()` generated by Clang embedded in Swift and standalone Clang.

Reply:
Done.